src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/SchemaGenerator.cs

```diff
@@ -249,7 +250,7 @@ private void ApplyMemberMetadata(OpenApiSchema schema, Type type, MemberInfo mem
                 schema.Reference = null;
             }
 
-            if (schema.Reference == null)
+            if (schema.Reference == null && memberInfo != null)
```

Comment:
I believe this additional check is unnecessary as the enclosing method `ApplyMemberMetadata` is only called on the condition that memberInfo is not null.